packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -121,7 +121,7 @@ const DropdownListBase = Skinnable(
 			 *
 			 * @type {('large'|'medium'|'small')}
 			 */
-			width: PropTypes.oneOf(['large', 'medium', 'small'])
+			width: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge'])
```

Comment:
Need to update the docs section, even though it's not used.  I'll grab it.  Maybe we'd be better off removing it but I don't know.